project/cmake/modules/FindVAAPI.cmake

```diff
@@ -29,6 +29,10 @@ elseif(VAAPI_INCLUDE_DIR AND EXISTS "${VAAPI_INCLUDE_DIR}/va/va_version.h")
   unset(vaapi_version_str)
 endif()
 
+if(VAAPI_VERSION_STRING VERSION_LESS 0.38)
+message(FATAL_ERROR "VAAPI minimum version 0.38 not found")
+endif()
+
```

Comment:
As explained some days ago, it's covered by the lines below where VAAPI_FIND_VERSION is set to 0.38. That variable is standard var read by find_package_handle_standard_arg which does the comparison and bails out if the version doesn't match. After wsnipex change, you can force vaapi now with -DENABLE_VAAPI=ON. Then if the lib is not found or the version is wrong, it fails. The default is ENABLE_VAAPI=AUTO. In that case it just compiles without vaapi. But the version is checked in both cases.